pkg/k8s/watchers/endpointsynchronizer.go

```diff
@@ -171,6 +180,7 @@ func (epSync *EndpointSynchronizer) RunK8sCiliumEndpointSync(e *endpoint.Endpoin
 								// label based selection for CiliumEndpoints.
 								Labels: pod.GetObjectMeta().GetLabels(),
 							},
+							Status: *mdl,
```

Comment:
I don't think this will work as expected. Unless we remove the subresource from the CRD itself, kube-apiserver will ignore this field and we can only update it via `UpdateStatus`.

Reply:
hmm I think you are right, converted this to draft to think about it more...

Reply:
Removed.